validator/evaluation/scoring.py

```diff
@@ -44,19 +44,25 @@
 logger = get_logger(__name__)
 
 
-def get_task_work_score(task: MiniTaskWithScoringOnly | TextRawTask | ImageRawTask) -> float:
+def get_task_work_score(task: MiniTaskWithScoringOnly) -> float:
     """Calculate work score for a task based on hours and model size."""
     assert task.hours_to_complete > 0, "Hours to complete must be positive"
     assert task.model_id, "Model ID must be present"
 
     hours = task.hours_to_complete
-    model = task.model_id
-    model_size = re.search(r"(\d+)(?=[bB])", model)
-    model_size_value = min(8, int(model_size.group(1)) if model_size else 1)
+
+    if getattr(task, 'model_params_count', 0) > 0:
```

Comment:
```suggestion
    if task.model_params_count:
```
This should be enough right? Unless we expect negative values (but that should be in the model validation then)